pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeterresource/create.yaml

```diff
@@ -12,11 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1alpha1
+apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1beta1
 kind: AccessContextManagerServicePerimeterResource
 metadata:
   name: accesscontextmanagerserviceperimeterresource-${uniqueId}
 spec:
   perimeterNameRef:
-    name: accesscontextmanagerserviceperimeter-${uniqueId}
-  resource: projects/${projectNumber}
+    # Referencing a resource name leads to recursive reference and KCC does not support the feature for now.
+    # Please use external reference instead.
```

Comment:
No need? Testdata is not for customers so we don't need to give instructions about how to use this resource.